src/docfx/lib/path/PackageUrl.cs

```diff
@@ -14,23 +15,26 @@ namespace Microsoft.Docs.Build
     /// The commit-sh can be any tag, sha, or branch. The default commit-ish is master.
     /// </summary>
     [JsonConverter(typeof(ShortHandConverter))]
-    internal readonly struct PackageUrl
+    internal class PackageUrl : IEquatable<PackageUrl>
```

Comment:
it does not need to implement `IEquatable`

Reply:
Already did in another PR